core/common/src/AuthorizationClient.ts

```diff
@@ -8,10 +8,15 @@
 
 import { AccessToken } from "@itwin/core-bentley";
 
-/** Interface to provide authorization information
- * @beta
+/** Provides [authorization](https://developer.bentley.com/apis/overview/authorization/#authorizesinglepageapplicationsspaanddesktopmobileapplicationsnative).
```

Comment:
I think this should say something more general about authorization for providing access to APIs as it doesn't necessarily need to be our (Bentley's) authorization provider that is used. I can take a better pass at this when back from holiday.

Reply:
Please do.

Reply:
starting on it now...

Reply:
Still working on this? Want me to give it a shot instead?

My change was motivated by the fact that people are going to want to know what the heck they should supply for this. Can they reasonably be expected to supply their own implementation instead of one of ours?

Reply:
I made changes. I'm going to resolve this comment later today unless you come back with further input.

Reply:
Time's up.